code/modules/client/preference_setup/general/03_body.dm

```diff
@@ -514,6 +542,52 @@ var/global/list/valid_bloodtypes = list("A+", "A-", "B+", "B-", "AB+", "AB-", "O
 				pref.organ_data[organ] = "mechanical"
 		return TOPIC_REFRESH
 
+	else if(href_list["reset_organs"])
+		reset_limbs()
+
+		return TOPIC_REFRESH
+
+	else if(href_list["shell"])
+		var/shellection = input(user, "What species shall you mimic?") as null|anything in list("Humanity","Vaurcae","Unathi","Tajara","Dionaea","Skrell")
+		if(!shellection) return
+
+		var/shell_type
+		switch(shellection)
+			if("Humanity")
+				shell_type = "Human"
+				mob_species.tail = null
+				mob_species.tail_animation = null
+			if("Vaurcae")
+				shell_type = "Vaurca"
+				mob_species.tail = null
+				mob_species.tail_animation = null
+			if("Unathi")
+				shell_type = "Unathi"
+				mob_species.tail = "sogtail"
+				mob_species.tail_animation = 'icons/mob/species/unathi/tail.dmi'
+			if("Tajara")
+				shell_type = "Tajara"
+				mob_species.tail = "tajtail"
+				mob_species.tail_animation = 'icons/mob/species/tajaran/tail.dmi'
+			if("Dionaea")

```

Comment:
Why not? I've excluded no species. Let the loremin have feedback, but I don't really see any particular reason to exclude it. Shells are noticeably poor mimics anyways.